### PR TITLE
[AIRFLOW-2429] Fix operators folder flake8 error

### PR DESCRIPTION
**Pull request by feng-tao**
Make sure you have checked _all_ steps below.

### JIRA
- [X] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2429
    - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a JIRA issue.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
  1. fix operators folder flake8 error
  2. set max-line-length to 120 for flake8 setting

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality
- [ ] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by feng-tao** [comment]
PTAL @kaxil  @Fokko

**Comment by kaxil** [comment]
Thanks @feng-tao . I will review and test this by end of tomorrow

**Comment by feng-tao** [comment]
Thanks @kaxil

**Comment by kaxil** [comment]
Can you also fix this:

```
./__init__.py:22:1: F401 'airflow.models.BaseOperator' imported but unused
./python_operator.py:195:91: E501 line too long (92 > 90 characters)
./python_operator.py:215:91: E501 line too long (95 > 90 characters)
./python_operator.py:216:91: E501 line too long (91 > 90 characters)
./python_operator.py:233:91: E501 line too long (93 > 90 characters)
./python_operator.py:249:91: E501 line too long (91 > 90 characters)
./python_operator.py:321:91: E501 line too long (94 > 90 characters)
./python_operator.py:347:91: E501 line too long (116 > 90 characters)
./python_operator.py:349:91: E501 line too long (120 > 90 characters)
./python_operator.py:359:91: E501 line too long (112 > 90 characters)
./python_operator.py:363:91: E501 line too long (99 > 90 characters)
./python_operator.py:371:91: E501 line too long (106 > 90 characters)
./python_operator.py:374:91: E501 line too long (94 > 90 characters)
./docker_operator.py:35:91: E501 line too long (100 > 90 characters)
./docker_operator.py:36:91: E501 line too long (94 > 90 characters)
./docker_operator.py:61:91: E501 line too long (97 > 90 characters)
./docker_operator.py:66:91: E501 line too long (100 > 90 characters)
./docker_operator.py:68:91: E501 line too long (99 > 90 characters)
./docker_operator.py:72:91: E501 line too long (92 > 90 characters)
./docker_operator.py:77:91: E501 line too long (100 > 90 characters)
./docker_operator.py:85:91: E501 line too long (94 > 90 characters)
./docker_operator.py:91:91: E501 line too long (97 > 90 characters)
./docker_operator.py:219:91: E501 line too long (100 > 90 characters)
```

File: setup.cfg [hunk 1]
```diff
@@ -33,5 +33,8 @@ all_files = 1
 [upload_sphinx]
 upload-dir = docs/_build/html
 
+[flake8]
+max-line-length = 120
```
Comment (kaxil) [inline setup.cfg]: Any reason for changing the `max-line-length`?

Comment (Fokko) [reply setup.cfg]: I guess there are a LOT of violations. I think it s by default 79, which is a bit on the low side, 120 is quite hight, maybe set it to 100?

Comment (kaxil) [reply setup.cfg]: Yes, The default for flake8 is 79 but for our project it has been set as 90 : https://github.com/apache/incubator-airflow/blob/c7a472ed6b0d8a4720f57ba1140c8cf665757167/tox.ini#L30

Comment (Fokko) [reply setup.cfg]: Ah thanks @kaxil. Lets set it to 90 then! :)

Comment (feng-tao) [reply setup.cfg]: thanks for the feedback @Fokko , @kaxil . I update the pr with 90. PTAL.

**Comment by kaxil** [comment]
@feng-tao I request you to not open/close the PR to trigger the travis CI. If you want to run the build again please run travis CI on your cloned repo as Travis would be free and give us the link for the build.

We have been warned by Apache Infra team regarding the consumption of CPU time as they pay for it.

Thanks.

**Comment by kaxil** [comment]
Can you please rebase against the master? The CI was failing for unrelated tests which is now fixed.

**Comment by feng-tao** [comment]
@kaxil , just rebased, the master is still broken.

**Comment by kaxil** [comment]
:( Hopefully it should be fixed now that the commit that broke the master in the first place is reverted.

**Comment by feng-tao** [comment]
@kaxil ,  I rebase with master again, now the error seems to fade away for most of the orms.

**Review by Fokko — APPROVED** [review]
Thanks @feng-tao awesome work!

**Pull request closed without merging**